spec/controllers/decidim/action_delegator/admin/delegations_controller_spec.rb

```diff
@@ -18,7 +18,28 @@ module Admin
           sign_in user
         end
 
+        describe "#index" do
+          it "authorizes the action" do
+            expect(controller).to receive(:allowed_to?).with(:index, :delegation, {})
```

Comment:
I would rewrite these expectations to the more comprehensive:

```suggestion
            expect(controller.allowed_to? :index, :delegation).to eq true
```